src/components/calculator.js

```diff
@@ -1,65 +1,63 @@
-/* eslint-disable react/prefer-stateless-function, jsx-a11y/click-events-have-key-events, react/no-unused-state, max-len, react/jsx-one-expression-per-line, jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable react/prefer-stateless-function, jsx-a11y/click-events-have-key-events, react/no-unused-state, max-len, react/jsx-one-expression-per-line, jsx-a11y/control-has-associated-label, import/prefer-default-export */
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import './calculator.css';
-// import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.operationHandler = this.operationHandler.bind(this);
-    this.state = {
-      total: 0,
-      next: 0,
-      operation: null,
-    };
-  }
+export const Calculator = () => {
+  // constructor(props) {
+  //   super(props);
+  //   this.operationHandler = this.operationHandler.bind(this);
+  //   this.state = {
+  //     total: 0,
+  //     next: 0,
+  //     operation: null,
+  //   };
+  // }
```

Comment:
- [ ] Let's remove chunks of code that are not used anymore. We can always use git history if we need to review old pieces of code and this will help our repo to be kept professional. [You can read more about best practices for JavaScript here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)